server/controllers/user.js

```diff
@@ -4,7 +4,7 @@ const UsersController = {
   Create: async (req, res) => {
     const user = new User(req.body);
```

Comment:
Excellent job as an overall. Have you tried to save a user in the db with this line of code?